src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -79,6 +85,20 @@ def load_arguments(self, _):
         configured_default=None
     )
 
+    snapshot_name_arg_type = CLIArgumentType(
+        options_list=['--snapshot-name', '-s'],
+        type=str,
+        help='Name of the App Configuration snapshot.',
+        configured_default=None
+    )
+
+    snapshot_filter_arg_type = CLIArgumentType(
+        options_list=['--filters'],
+        validator=validate_snapshot_filters,
+        nargs='+',
+        help='Space-separated key and label filters used to build an app configuration snapshot. These values should escaped JSON objects.'
```

Comment:
> These values should escaped JSON objects

These values should **be** escaped JSON objects

Reply:
My bad. Thanks! This has been fixed.